src/data.rs

```diff
@@ -92,6 +140,10 @@ macro_rules! impl_try_from {
     impl<'s> TryFrom<Local<'s, $source>> for Local<'s, $target> {
       type Error = DataError;
       fn try_from(l: Local<'s, $source>) -> Result<Self, Self::Error> {
+        #[allow(dead_code)]
+        fn cast<T>(l: Local<$source>) -> Local<T> {
+          unsafe { transmute(l) }
+        }
```

Comment:
Remove?

Reply:
It's not actually dead code. I've added a comment.